scripts/setupvars/setupvars.sh

```diff
@@ -146,8 +146,8 @@ check_python_version () {
     fi
 } 
 
-python_version_to_check="$python_version"
-if [ -z "$python_version" ]; then
+python_version_to_check="$python_version_major"
+if [ -z "$python_version_to_check" ]; then
```

Comment:
looks like `python_version_major` is not defined yet here
maybe we need to revert the change?